di_website/common/constants.py

```diff
@@ -28,6 +28,7 @@
     'anchor',
     'footnote'
 ]
+ALT_MAX_RELATED_LINKS = 2
```

Comment:
If this is only going to be used for a specific page type, add it somewhere close to wherever it's being used. It doesn't look useful here.